chart/templates/deployment.yaml

```diff
@@ -0,0 +1,121 @@
+    heritage: Helm
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      app: poc-acapy-multitenant
+      release: poc-acapy-multitenant
+  template:
+    metadata:
+      annotations:
+        # needs to be updated I suppose
+        iam.amazonaws.com/role: arn:aws:iam::715299534867:role/gid-dev-poc-acapy-multitenant
+      labels:
+        app: poc-acapy-multitenant
+        release: poc-acapy-multitenant
+    spec:
+      containers:
+        - name: poc-acapy-multitenant
+          # needs to be updated with published image value
+          # image: "715299534867.dkr.ecr.us-east-1.amazonaws.com/poc-acapy-multitenant:2a333a454c04f014aa02895249b2a98d4d924b78" 
+          imagePullPolicy: IfNotPresent
+          ports:
+            - name: mainport
+              containerPort: 9836
+          resources:
+            limits:
+              cpu: 500m
+              memory: 768Mi
+            requests:
+              cpu: 250m
+              memory: 192Mi
+          # should readinessProbe and startupProbe be present? 
+          readinessProbe:
+            httpGet:
+              path: poc-acapy-multitenant/health/ready
+              port: 8800
+            initialDelaySeconds: 15
+            periodSeconds: 10
+            failureThreshold: 25
+          startupProbe:
+            httpGet:
+              path: poc-acapy-multitenant/health/alive
+              port: 8800
+            initialDelaySeconds: 15
+            periodSeconds: 10
+            failureThreshold: 10
+          envFrom:
+            - configMapRef:
+                name: shared-config
+            - configMapRef:
+                name: poc-acapy-multitenant
+          env:
+          - name: ACAPY_ENDPOINT:
+            value: 'http://172.17.0.1:8040' # acapy endpoint should be the actual url on which other agents can access him
+          - name: ACAPY_INBOUND_TRANSPORT:
+            value: 'http 0.0.0.0 8040'
+          - name: ACAPY_NO_LEDGER:
+            value: 'true'
+          - name: ACAPY_OUTBOUND_TRANSPORT:
+            value: 'http'
+          - name: ACAPY_ADMIN:
+            value: '0.0.0.0 8041'
+          - name: ACAPY_ADMIN_INSECURE_MODE:
+            value: 'true'
+          - name: ACAPY_WALLET_TYPE:
+            value: 'askar'
+          - name: ACAPY_AUTO_PROVISION:
+            value: 'true'
+          - name: ACAPY_TRACE_TARGET:
+            value: 'log'
+          - name: ACAPY_WALLET_STORAGE_TYPE:
+            value: 'postgres_storage'
+          - name: ACAPY_WALLET_STORAGE_CONFIG:
+            value: '{"url":"172.17.0.1:5437"}' # same as for ACAPY_ENDPOINT
+          - name: ACAPY_WALLET_STORAGE_CREDS:
+              # update with reald db values
+            value: '{\"account\":\"db_user_name\",\"password\":\"db_password\",\"admin_account\":\"admin_db_user_name\",\"admin_password\":\"admin_db_password\"}'
```

Comment:
this should be a secret not an env

Reply:
how to do that?

Reply:
You had examples in for chime

Reply:
example
` - name: PUBNUB_SUBSCRIBE_KEY
            valueFrom:
              secretKeyRef:
                key: pubnub-subscribe-key
                name: messaging-service`